_quarto.yml

```diff
@@ -1,6 +1,16 @@
+date: today
+date-format: iso
+engine: julia
+julia:
+  exeflags: ["--project", "--threads=auto"]
+execute:
+  cache: true
```

Comment:
I don't think this has an effect with current versions of the julia engine

Reply:
I don't think so either, but I was copying over as many parameters as I could